src/rebar_prv_shell.erl

```diff
@@ -129,6 +129,8 @@ setup_shell() ->
     end.
 
 setup_paths(State) ->
+    %% Change working directory for the project root
+    file:set_cwd(rebar_state:dir(State)),
```

Comment:
no. If this needs to be done the issue is that something else in the build tool is switching directories and that's where the bug is to begin with.